src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Profile Pages</title>
+	<link rel="stylesheet" href="/css/main.css" type"text/css">
+
+	<style>
+		#chat {
+			background-color: white;
+			height: 500px;
+			overflow-y: scroll
+		}
+	</style>
+
+  <script>
+    // scroll the chat div to the bottom
+    function scrollChat() {
+      var chatDiv = document.getElementById('chat');
+      chatDiv.scrollTop = chatDiv.scrollHeight;
+    };
+  </script>
+</head>
+<body onload="scrollChat()">
+
+  <nav>
+    <a id="navTitle" href="/">CodeU Chat App</a>
+    <a href="/conversations">Conversations</a>
+      <% if (request.getSession().getAttribute("user") != null) { %>
+    <a>Hello <%= request.getSession().getAttribute("user") %>!</a>
+    <% } else { %>
+      <a href="/login">Login</a>
+    <% } %>
+    <a href="/about.jsp">About</a>
+  </nav>
+
```

Comment:
It might be useful to add a link to your own profile page in the Nav bar if you are logged in (for the other pages too) and it would also be useful for manually testing too. Or add a todo comment for this since it's not a priority right now.